pipelines/LSST/DC2/ci_imsim.yaml

```diff
@@ -0,0 +1,26 @@
+description: DRP specialized for ci_imsim test package.
+imports:
+  - $DRP_PIPE_DIR/ingredients/LSST/DC2/DRP.yaml
+tasks:
+  deblend:
+    class: lsst.pipe.tasks.deblendCoaddSourcesPipeline.DeblendCoaddSourcesMultiTask
+    # Significantly limit the number of sources to deblend, which saves time
+    # in both deblending and measurement
+    config:
+      # Only run the deblender on a small subset of blended parents
+      multibandDeblend.useCiLimits: true
```

Comment:
This is a mess over in `ci_imsim`: the options do indeed have a default of `False` in the argument parser over in `ci_imsim_run.py`, but then the `ups/eupspkg.cfg.sh` script that runs it in CI explicitly passes `--config-limit-deblend` on the command-line making them `True`, so that's the behavior most people are used to.

I don't think I can fix this without fixing the inconsistent behavior of `ci_imsim` itself, and I don't want to do that here; I'll open a ticket, because that's a problem in its own right.

Reply:
[DM-33459](https://jira.lsstcorp.org/browse/DM-33450)